src/ArticleListView.m

```diff
@@ -857,8 +856,10 @@ -(void)setOrientation:(NSInteger)newLayout
 {
 	isChangingOrientation = YES;
 	tableLayout = newLayout;
+	splitView2.autosaveName = nil;
```

Comment:
Can you please remove the autosaveName from Interface Builder too? That way a future contributor won't look at IB and get confused as to why the autosaveName isn't being used 😄 

![screen shot 2017-09-20 at 10 40 23](https://user-images.githubusercontent.com/799345/30621731-217290d2-9df0-11e7-8d9f-667d9295c543.png)

Reply:
Done.